### PR TITLE
syncing

### PR DESCRIPTION
**Pull request by sfshaza2**
as it says

### CODE REVIEW

**Comment by domesticmouse** [comment]
@parlough can I get some guidance on what I need to do with `startup_namer`'s `pubspec.yaml` such that it does the right thing in the diff?

**Comment by domesticmouse** [comment]
I've committed what the code snippet tool has updated the highlight diff as, but it's not what I want to see. Halp.

/cc @khanhnwin

File: src/get-started/codelab.md [hunk 1]
```diff
@@ -244,10 +244,19 @@ as well as many other open source packages, on [pub.dev][].
     ```diff
     --- step1_base/pubspec.yaml
     +++ step2_use_package/pubspec.yaml
-    @@ -9,4 +9,5 @@
+    @@ -24,13 +23,5 @@
+    -# To automatically upgrade your package dependencies to the latest versions
```
Comment (domesticmouse) [inline src/get-started/codelab.md:248]: This is all kinds of wrong. Gah.

Comment (domesticmouse) [reply src/get-started/codelab.md:248]: I think I have a fix in flight for this.

**Comment by domesticmouse** [comment]
This PR is blocked on https://github.com/flutter/codelabs/pull/1003 being landed and rolled into this PR.

**Review by domesticmouse — APPROVED** [review]

**Review by parlough — APPROVED** [review]
Thanks for adjusting the rebuild script @domesticmouse!

**Comment by domesticmouse** [comment]
> Thanks for adjusting the rebuild script @domesticmouse! I'd love to learn more about the system :)

Throw some time on my calendar early next week or the following week and I'll answer all your questions 🤓

**Comment by khanhnwin** [comment]
Looks like this has already been landed!

**Comment by domesticmouse** [comment]
> Looks like this has already been landed!

Parker helped me get this over the finishing line 🥳

**Pull request merged**